### PR TITLE
add support for safari browser

### PR DESCRIPTION
**Pull request by joaoedezio**
This commit solve the compatibility with safari browser, using webkitAudioContext, this is implemented by:
this.context = new (window.AudioContext || window.webkitAudioContext)(); //support for safari and safari in iOS

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -15911,13 +15911,14 @@ var MicRecorder = function () {
     value: function start() {
       var _this2 = this;
 
-      this.context = new AudioContext();
+      // this.context = new AudioContext();
```
Comment (djalmaaraujo) [inline dist/index.js]: This comment is useless :)

File: dist/index.js [hunk 2]
```diff
@@ -15911,13 +15911,14 @@ var MicRecorder = function () {
...
+	  this.context = new (window.AudioContext || window.webkitAudioContext)(); //support for safari and safari in iOS
       this.config.sampleRate = this.context.sampleRate;
       this.lameEncoder = new Encoder(this.config);
 
       return new Promise(function (resolve, reject) {
         navigator.mediaDevices.getUserMedia({ audio: true }).then(function (stream) {
-          _this2.addMicrophoneListener(stream);
+      		_this2.addMicrophoneListener(stream);
```
Comment (djalmaaraujo) [inline dist/index.js]: You changed the 2 spaces to 4 spaces here.

**Comment by djalmaaraujo** [comment]
@buzinas Can you take a look also, since you guys are using this?

**Comment by buzinas** [comment]
You should change `src/mic-recorder.js` instead of `dist/index.js` (which is a generated file).

**Comment by djalmaaraujo** [comment]
@joaoedezio ☝️

**Comment by joaoedezio** [comment]
sorry for that, I will do this and reply soon

**Comment by buzinas** [comment]
Closing for inactivity.

**Pull request closed without merging**